src/styles/colors.ts

```diff
@@ -0,0 +1,44 @@
+import { ThemesEnum } from '../interfaces/theme.enum';
+
+export const getColors = (theme: ThemesEnum): Colors => ({
+  ...baseColors,
+  ...(theme === ThemesEnum.light && darkGray)
+});
+
+export type Colors = {
+  [key in keyof typeof baseColors]: string;
+};
+
+const baseColors = {
+  blue: '#007AFF',
+  blue10: 'rgba(0, 122, 255, 0.1)',
+  orange: '#FF5B00',
+  orange10: 'rgba(255, 122, 0, 0.1)',
```

Comment:
let's use dependant alpha colors. (ex: `orange10: hexa(orange, 0.1);`);
When designers will change `orange color, we will change it in single place